### PR TITLE
ref(avatar): Move Avatar and Participant view to base/participants

### PR DESCRIPTION
**Pull request by hristoterezov**

### CODE REVIEW

File: react/features/base/participants/components/ParticipantView.native.js [hunk 1]
```diff
@@ -1,14 +1,14 @@
 import React, { Component } from 'react';
 import { connect } from 'react-redux';
 
+import { getParticipantById } from '../functions';
```
Comment (yanas) [inline react/features/base/participants/components/ParticipantView.native.js]: Move this import to the in-feature imports section and arrange alphabetically within that section.

File: react/features/base/participants/components/styles.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    /**
+     * Avatar style.
+     */
+    avatar: {
+        flex: 1,
+        width: '100%'
+    },
+
+    /**
+     * ParticipantView style
```
Comment (yanas) [inline react/features/base/participants/components/styles.js]: dot missing at the end of the sentence

File: react/features/base/participants/index.js [hunk 3]
```diff
@@ -1,5 +1,6 @@
 export * from './actions';
 export * from './actionTypes';
+export * from './components';
 export * from './constants';
 export * from './functions';
```
Comment (yanas) [inline react/features/base/participants/index.js]: Arrange exports alphabetically.

**Comment by lyubomir** [comment]
Thank you very much, @hristoterezov!

**Pull request merged**